src/Tizen.NUI/src/public/WindowEvent.cs

```diff
@@ -703,6 +749,17 @@ private void OnTransitionEffect(IntPtr window, int state, int type)
             return;
         }
 
+        private void OnKeyboardRepeatSettingsChanged()
+        {
+            KeyboardRepeatSettingsChangedArgs e = new KeyboardRepeatSettingsChangedArgs();
+
+            if (keyboardRepeatSettingsChangedHandler != null)
+            {
+                keyboardRepeatSettingsChangedHandler(this, e);
```

Comment:
if the Args is removed then this should be like this : keyboardRepeatSettingsChangedHandler(this, null);

Reply:
Okay, Done.